### DISCUSSION TITLE
Should we remove `--pf` styles from Base classes?

### DISCUSSION THREAD

**Discussion by bennypowers** [RedHat-UX/red-hat-design-system#578] [Polls]
A number of pfe elements in the 1:1 milestone use `--pf-*` tokens in their base classes, which means that their inheriting RHDS versions also pack along those tokens, even though their stylesheets typically overwrite them entirely with RHDS tokens. We typically redefine the built-in css properties those tokens assign to in RHDS, rather than aliasing the pf tokens with RHDS tokens, so the often times the PatternFly tokens end up as dead weight.

Should we remove PatternFly styles from base classes, moving them to the `pfe-*.scss` stylesheets? There would be no visual changes in either project, but RHDS elements would have a reduction in 'bundle size'.

**Comment by heyMP** [comment]
These always scare me because of the possible downstream issues but my knee jerk reaction is to say get 'em outta there. 👍

**Reply by bennypowers** [reply]
it's important to point out that if we implement this proposal as intended, then users who do this will see no difference whatsoever:

```html
<script type="module">
// I haven't changed, only the base class has changed.
// All my CSS still exists, except now it exists exclusively on PfeJazzHands,
// and not at all on BaseJazzHands
import '@patternfly/pfe-jazz-hands';
</script>
<pfe-jazz-hands></pfe-jazz-hands>

**Reply by heyMP** [reply]
Yeah, totally.  I fully support this rule 👍

**Comment by bennypowers** [comment]
Closing because we did it
